test/integration/utility.h

```diff
@@ -110,10 +110,10 @@ class ConnectionStatusCallbacks : public Network::ConnectionCallbacks {
   bool closed() const { return closed_; }
 
   // Network::ConnectionCallbacks
-  void onEvent(uint32_t events) override {
-    closed_ |= (events & Network::ConnectionEvent::RemoteClose ||
-                events & Network::ConnectionEvent::LocalClose);
-    connected_ |= events & Network::ConnectionEvent::Connected;
+  void onEvent(Network::ConnectionEvent event) override {
+    closed_ |= (event == Network::ConnectionEvent::RemoteClose ||
+                event == Network::ConnectionEvent::LocalClose);
+    connected_ |= (event == Network::ConnectionEvent::Connected);
```

Comment:
It looks like the standard concretely guarantees that false is 0 and true is 1, so I think it's safe from UB. I think ye olde time C (before C99) only guaranteed that true expressions were non-zero. I guess it still works in that case but looks odd.

I'm OK with leaving it as is.